src/TNBase.DataStorage/Service/ServiceLayer.cs

```diff
@@ -65,36 +65,40 @@ public List<Listener> GetOnlineOnlyListenersOrderedBySurname()
             return context.Listeners.Where(x => !x.Status.Equals(ListenerStates.DELETED) && x.OnlineOnly).OrderBy(x => x.Surname).ToList();
         }
 
-        public List<Listener> GetNextWeekBirthdays()
+        public List<Listener> GetUpcomingBirthdays(DateTime fromDate, DateTime toDate)
         {
             var list = context.Listeners.ToList().Where(x =>
                 x.HasBirthday &&
                 !x.Status.Equals(ListenerStates.DELETED))
             .ToList();
 
-            DateTime nowDate = DateTime.Now.Date;
-            DateTime weekDate = DateTime.Now.AddDays(1).Date;
+
+            return list.Where(x =>
+            {
+                var nextBirthdate = x.NextBirthdayDate;
+                return nextBirthdate >= fromDate && nextBirthdate <= toDate;
+            }).ToList();
+        }
+
+        public void GetUpcomingBirthdayDates(out DateTime fromDate, out DateTime toDate)
```

Comment:
As this method's sole purpose is to return dates, I would prefer if it would use a return object with `from` and `to` dates rather than using out parameters.